webapp/channels/src/components/sidebar/sidebar_category/sidebar_category.tsx

```diff
@@ -270,30 +269,21 @@ export default class SidebarCategory extends React.PureComponent<Props, State> {
         } else if (category.type === CategoryTypes.DIRECT_MESSAGES) {
             const addHelpLabel = localizeMessage('sidebar.createDirectMessage', 'Create new direct message');
 
-            const addTooltip = (
-                <Tooltip
-                    id='new-group-tooltip'
-                    className='hidden-xs'
-                >
-                    {addHelpLabel}
```

Comment:
we missed this one

Reply:
oops, i'll work on it asap

Reply:
Please also remember to screen shot the usage as it will make QA easy

Reply:
![image](https://github.com/user-attachments/assets/a8d4bd2e-92c9-49ed-b8b1-f7144e74e57d)